exercises/ch04/lessons_learned.md

```diff
@@ -0,0 +1,24 @@
+# Lessons Learned
+- If you see a function with the shape:
+
+  ```javascript
+  b => someFunction(a, b)
+  ```
+
+  where b is a curried function, that's equivalent to:
+
+  ```javascript
+  someFunction(a)
+  ```
+
+- If you see a function call like:
+
+  ```javascript
+  b.someFunction(a)
+  ```
+
+  that's equivalent to calling the curried function
+
+  ```javascript
+  someFunction(a)
+  ```
```

Comment:
Not really. This is only true if `someFunction` doesn't rely on `this`. Also has little do to with currying I am afraid, so this is off-topic for Chapter 4.